### ISSUE TITLE
Color contrast on "Continue Reading" links

### ISSUE THREAD

**Issue by phucduong86** [phucduong86/phucduong86.github.io#8]
Does not meet minimum requirements

**Comment by phucduong86** [comment]
This issue as been resolved.
